src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,12 @@
+package codeu.chat.common;
+import codeu.chat.util.Time;
+
+public final class ServerInfo {
```

Comment:
Will be nice to add javadoc comments for all public classes and public methods at least.